.github/workflows/release.yml

```
@@ -11,12 +11,12 @@ jobs:
     if: github.repository_owner == 'voxpupuli'
     steps:
       - uses: actions/checkout@v4
-      - name: Install Ruby 3.1
+      - name: Install Ruby 3.3
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '3.1'
+          ruby-version: '3.3'
```

In another place I've been lazy and just used Ruby 3. Is that something we want here as well?
```suggestion
          ruby-version: '3'
```

---

I'm not sure. I don't know under which conditions they will bump that version. I only bump the version in the release action when we've the same version in a test and it passes.